OpenRA.Mods.TS/Effects/AnimatedBeaconSprite.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
```

Comment:
I can't comment on the functionality, but you should use the updated copyright header here. It's also 2016 :)